packages/core/src/types.ts

```diff
@@ -76,3 +77,12 @@ export interface OutboundPackage {
   endpoint?: string
   connectionId?: string
 }
+
+// Metadata type for `_internal/indyCredential`
+export interface indyCredential {
```

Comment:
I think we should make interfaces always uppercase.

Maybe also make them a bit more elaborate?

```ts
export interface IndyCredentialMetadata

// and

export interface IndyCredentialRequestMetadata
```